docs/firstobject.md

```diff
@@ -105,11 +107,45 @@ We will come back to this later.
 </div>
 </details>
 
-Make sure you select `Photograph` as `Media Type` and all the fields with a red `*` are filled up. Press `Next: Linked data`. That will load the next step inline.
+Make sure you select `Photograph` as `Media Type` and all the fields with a red `*` are filled up. Press `Next: Linked data`. That will load the next step inline. 
 
 Note: Don't press `Save`, Don't press `Preview`. Why? You are not ready yet!
 
+### Linked Data
+
 ![Ingest Step2](../imgs/ingest-step2.jpg)
 
-This Step showcases some of the Autocomplete Linked data Webform elements we built for Archipelago. We truly believe in Wikidata as a open, honest, source of Linked Open Data and also one where you can contribute back. But we also have LoC autocompletes and Getty. More to come!
+As the name of this step suggests; on this page you will be adding all your Linked Data for your Digital Object. This Step showcases some of the Autocomplete Linked data Webform elements we built for Archipelago. We truly believe in Wikidata as an open, honest, source of Linked Open Data and also one where you can contribute back. But we also have LoC autocompletes and Getty.
```

Comment:
I would suggest here all your Linked Data "elements" or "values". Also make sure that Linked Data is always starts with Capital.